dsb-gui/src/main/java/com/spectralogic/dsbrowser/gui/components/ds3panel/Ds3PanelPresenter.java

```diff
@@ -563,13 +566,8 @@ public void ds3DeleteObject(final boolean isButtonEvent) {
                 alert.showAlert(resourceBundle.getString("noFiles"));
             }
         } else if (values.stream().map(TreeItem::getValue).anyMatch(value -> value.getType() == Ds3TreeTableValue.Type.Directory)) {
-            if (isButtonEvent) {
-                LOG.error("You can only recursively delete a folder.  Please select the folder to delete, Right click, and select \'Delete Folder...\'");
-                alert.showAlert(resourceBundle.getString("canRecursivelyDelete"));
-            } else {
-                LOG.info("Going to delete the folder");
-                DeleteService.deleteFolder(ds3Common, values, loggingService, resourceBundle);
-            }
+            values.stream().map(TreeItem::toString).forEach(itemString -> LOG.info("Delete folder " + itemString));
```

Comment:
This should use the logging statements formatted like: `LOG.info("Delete folder {}", itemString)`